cmake/modules/AddSwift.cmake

```diff
@@ -548,6 +548,13 @@ function(_add_swift_host_library_single target)
       target_compile_definitions(${target} PRIVATE
         _WINDLL)
     endif()
+  endif()  
+  
+  # Double-check that we're not trying to build a dynamic library for WASM.
+  if(SWIFTLIB_SINGLE_SDK MATCHES WASM)
```

Comment:
This shouldn't happen right?  Because this is for the toolchain.

Reply:
It shouldn't, and didn't happen in my practice building it, was written more like an assertion here. I can remove it if you'd like.

Reply:
Please do, I think that minimizing the CMake logic is preferable.

Reply:
That's cleaned up now